OpenRA.Mods.Common/UtilityCommands/ActorStatsExport.cs

```diff
@@ -79,7 +80,9 @@ public static DataTable GenerateTable()
 							var burst = weapon.Burst;
 							var delay = weapon.BurstDelay;
 							var damage = warhead.Damage;
-							var damagePerSecond = (1000f / Game.Timestep) * (damage * burst) / (delay + burst * rateOfFire);
+
+							// Assumes the default game speed (25 ticks / sec)
+							var damagePerSecond = (1000f / 40) * (damage * burst) / (delay + burst * rateOfFire);
```

Comment:
It is certainly cleaner for the lobby code as well. Currently you are enforcing a hidden string comparison. If someone does not define a game speed named "Default", but maybe call it "Normal" like in the originals, it will also break.